lib/archethic/beacon_chain/subset.ex

```diff
@@ -330,4 +349,9 @@ defmodule ArchEthic.BeaconChain.Subset do
     }
     |> ValidationStamp.sign()
   end
+
+  def subscribe_for_beacon_updates(subset, node_public_key) do
```

Comment:
Can you add some typespec for public functions and maybe some documentation?

Reply:
definitely docs is must and also spec